cegs_portal/search/templates/search/v1/partials/_multi_experiment_index.html

```diff
@@ -83,4 +207,5 @@
     addDragListeners();
     addSelectListeners();
 </script>
+<script src="https://cdn.jsdelivr.net/npm/tw-elements/js/tw-elements.umd.min.js"></script>
```

Comment:
You can stick this in the actual `experiments.html` template, but also note that we already load it a particular way elsewhere: `<script src="{% static 'js/tw-elements/index.min.js' %}"></script>`

Reply:
Right. Thought there were issues with it before but turns out another error was causing the problem. Need to add this back.